.doc_gen/metadata/bedrock_metadata.yaml

```diff
@@ -17,14 +17,37 @@ bedrock_Hello:
       versions:
         - sdk_version: 3
           github: dotnetv3/Bedrock
-          sdkguide:
           excerpts:
             - description:
               snippet_tags:
                 - Bedrock.dotnetv3.BedrockActions.HelloBedrock
+    JavaScript:
+      versions:
+        - sdk_version: 3
+          github: javascriptv3/example_code/bedrock
+          excerpts:
+            - description:
+              snippet_files:
+                - javascriptv3/example_code/bedrock/hello.js
   services:
     bedrock: {ListFoundationModels}
 
+bedrock_GetFoundationModel:
+  title: Get details about an &BRlong; foundation model using an &AWS; SDK
+  title_abbrev: Get details about an &BR; foundation model
+  synopsis: get details about an &BR; foundation model.
+  languages:
+    JavaScript:
+      versions:
+        - sdk_version: 3
+          github: javascriptv3/example_code/bedrock
+          excerpts:
+            - description: Get details about a foundation model.
+              snippet_files:
+                - javascriptv3/example_code/bedrock/actions/getFoundationModel.js
```

Comment:
You've got the wrong filename here. Should be `get-foundation-model.js`

Reply:
Good catch